static/app/comps/Converter/convert.js

```diff
@@ -0,0 +1,42 @@
+const measures = [
+  {
+    unit: 'football field',
+    property: 'length',
+    factor: 109.7
```

Comment:
how do u feel about extending this to multiple targets? what if we used paperclips for length and weight and volume?????

Reply:
lgtm